doc/DISCLAIMER.md

```diff
@@ -0,0 +1,4 @@
+## YunoHost specific features
+
+* SSO and LDAP are configured.
+* Login secured by fail2ban
```

Comment:
Is it ? The install script seems to have no fail2ban conf in the end. Comment mentions that limesurvey already contains some brute forcing security, which is probably cool, but not a "yunohost specific feature" like the DISCLAIMER.md says ?

Reply:
Indeed
```suggestion
```